repo/packages/K/kubernetes/4/config.json

```diff
@@ -194,6 +194,29 @@
           "description": "Which proxy mode to use: 'userspace' (older) or 'iptables' (faster). If the iptables proxy is selected, regardless of how, but the system's kernel or iptables versions are insufficient, this always falls back to the userspace proxy.",
           "type": "string",
           "pattern": "^(userspace|iptables)$"
+        },
+        "pki": {
```

Comment:
the current version of dcos CLI enforces "required" for `pki.*` properties, even though `pki` itself is optional:
```sh
$ dcos package install kubernetes
In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.
We recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.
Note that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.

WARNING: The standard configuration launches an embedded etcd WHICH LOOSES
         THE COMPLETE CLUSTER STATE when the framework container restarts.
         For production follow http://docs.mesosphere.com/services/kubernetes/#install
         to configure kubernetes with a highly available, replicated etcd cluster.

Continue installing? [yes/no] yes
Error: missing required property 'apiserver-crt'.

Error: missing required property 'apiserver-key'.

Error: missing required property 'root-ca-crt'.

Error: missing required property 'service-accounts-key'.

Please create a JSON file with the appropriate options, and pass the /path/to/file as an --options argument.
```